pkgs/build-support/agda/default.nix

```
@@ -76,6 +77,7 @@ let
           find -not \( -path ${everythingFile} -or -path ${lib.interfaceFile everythingFile} \) -and \( ${concatMapStringsSep " -or " (p: "-name '*.${p}'") (extensions ++ extraExtensions)} \) -exec cp -p --parents -t "$out" {} +
           runHook postInstall
         '';
+        meta = if meta ? broken && meta.broken then meta // { hydraPlatforms = lib.platforms.none; } else meta;
```

Looks good, small suggestion which is only a refactor:

```suggestion
        meta = if meta.broken or false then meta // { hydraPlatforms = lib.platforms.none; } else meta;
```